src/container/srv_target.c

```diff
@@ -1523,9 +1539,16 @@ cont_child_aggregate(struct ds_cont_child *cont, uint64_t *sleep)
 		goto out;
 
 	epoch_range.epr_hi = epoch_max;
+
+	/* Not cross the rebuild epoch */
+	if (cont->sc_pool->spc_rebuild_epoch > epoch_range.epr_lo &&
+	    cont->sc_pool->spc_rebuild_epoch < epoch_range.epr_hi)
+		epoch_range.epr_hi = cont->sc_pool->spc_rebuild_epoch;
```

Comment:
So [rebuild_epoch + 1, epoch_max] isn't aggregated at the end. Probably we'd just regard rebuild epoch as a special snapshot and form a temporary snapshot array, then use this temporary snapshot array to get the aggregation ranges.